contentcuration/contentcuration/viewsets/user.py

```diff
@@ -415,10 +370,20 @@ def compose_annotations(self):
             .values_list("id", flat=True)
             .distinct()
         )
-        annotations["edit_count"] = SQCount(edit_channel_query, field="id")
-        annotations["view_count"] = SQCount(viewonly_channel_query, field="id")
-        return annotations
 
+        queryset = queryset.annotate(
+            editable_channels__ids=NotNullArrayAgg("editable_channels__id"),
+            view_only_channels__ids=NotNullArrayAgg(
+                "view_only_channels__id"
+            ),
```

Comment:
Looks like the above two annotations got re-added. I removed them because they didn't appear to be used anywhere

Reply:
Ah, probably a rebaseroo. Will remove.